### PR TITLE
feat: suggest adding a "node:" prefix for bare specifiers that look like built-in Node modules

### PR DESCRIPTION
**Pull request by dsherret**
Adds a better message on the CLI and quick fix in the LSP:

image

image

Closes https://github.com/denoland/deno/issues/17493

### CODE REVIEW

File: cli/tests/integration/lsp_tests.rs [hunk 1]
```diff
@@ -4465,21 +4459,108 @@ fn lsp_completions_node_specifier() {
...
+      "diagnostics": [
+        {
+          "range": {
+            "start": { "line": 0, "character": 15 },
+            "end": { "line": 0, "character": 19 }
+          },
+          "severity": 1,
+          "code": "import-prefix-missing",
+          "source": "deno",
+          "message": "Relative import path \"fs\" not prefixed with / or ./ or ../",
```
Comment (dsherret) [inline cli/tests/integration/lsp_tests.rs:4520]: I should update this message to be more explanatory as well.

**Comment by aapoalas** [comment]
Nice.

File: cli/tests/testdata/jsx/deno.lock [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "version": "2",
+  "remote": {
+    "http://localhost:4545/jsx/jsx-dev-runtime/index.ts": "183c5bf1cfb82b15fc1e8cca15593d4816035759532d851abd4476df378c8412"
+  }
+}
```
Comment (dsherret) [inline cli/tests/testdata/jsx/deno.lock:6]: This file has been created everytime I run all the tests for a few months so I'm just going to commit it.

**Comment by dsherret** [comment]
@aapoalas thanks for the quick fix suggestion!

**Comment by aapoalas** [comment]
> @aapoalas thanks for the quick fix suggestion!

Oh, I forgot I said that 😃 But good that it was helpful!

**Review by bartlomieju — APPROVED** [review]
LGTM, tested locally, works very nicely

**Pull request merged**